File: src/src_core/fmg/core/types/click/LeftDownResult.cs
using System.Collections.Generic;
using fmg.core.mosaic.cells;

namespace fmg.core.types.click {

public class LeftDownResult {
	public IList<BaseCell> needRepaint;

	public LeftDownResult()
	{
		needRepaint = new List<BaseCell>();
	}
}
}

File: src/src_core/fmg/core/types/click/LeftDownResult.java
package fmg.core.types.click;

import java.util.ArrayList;
import java.util.List;

import fmg.core.mosaic.cells.BaseCell;

public class LeftDownResult {
   public List<BaseCell> needRepaint;

   public LeftDownResult()
   {
      needRepaint = new ArrayList<BaseCell>();
   }
}
